requirements-common.txt

```diff
@@ -6,7 +6,7 @@ numpy < 2.0.0
 requests
 tqdm
 py-cpuinfo
-transformers >= 4.40.0  # Required for StarCoder2 & Llava, Llama 3.
+transformers >= 4.42.0  # Required for Gemma 2.
```

Comment:
4.42.0 is actually broken, should be 4.42.1 instead
https://github.com/huggingface/transformers/pull/31661

Reply:
Hi @estibi, if I understand correctly, the bug is not related to vLLM so it's not necesary for us to enforce `>= 4.42.1`. Is my understanding correct?

Reply:
Hi @WoosukKwon  I'm not sure how transformers library is used inside vLLM but 4.42.0 is failing during model.generate on Gemma 2 models.
https://huggingface.co/google/gemma-2-9b-it/discussions/8#667dbe146d330db13593b19e

Reply:
@estibi vLLM does not use `transformers` implementation of the models and only imports `PretrainedConfig` from `transformers`, therefore afaik this bug shouldn't affect us.

Reply:
Ok, thanks for the explanation 👍